apps/console/src/features/identity-providers/components/wizards/organization-enterprise/organization-enterprise-authentication-provider-create-wizard-content.tsx

```diff
@@ -0,0 +1,255 @@
+        } = props;
+
+        const { t } = useTranslation();
+
+        const [ idpList, setIdPList ] = useState<IdentityProviderListResponseInterface>({});
+        const [ isIdPListRequestLoading, setIdPListRequestLoading ] = useState<boolean>(undefined);
+
+        /**
+     * Loads the identity provider authenticators on initial component load.
+     */
+        useEffect(() => {
+
+            getIDPlist();
+        }, []);
+
+        /**
+     * Get Idp List.
+     */
+        const getIDPlist = (): void => {
+
+            setIdPListRequestLoading(true);
+
+            getIdentityProviderList(null, null, null)
+                .then((response) => {
+                    setIdPList(response);
+                })
+                .catch((error) => {
+                    handleGetIDPListCallError(error);
+                })
+                .finally(() => {
+                    setIdPListRequestLoading(false);
+                });
+        };
+
+        /**
+     * Check whether IDP name is already exist or not.
+     *
+     * @param value IDP name - IDP Name.
+     *
+     * @returns error msg if name is already taken.
+     */
+        const idpNameValidation = (value): string => {
+
+            let nameExist = false;
+
+            if (idpList?.count > 0) {
+                idpList?.identityProviders.map((idp) => {
+                    if (idp?.name === value) {
+                        nameExist = true;
+
+                    }
+                });
+            }
+            if (nameExist) {
+                return t("console:develop.features." +
+                "authenticationProvider.forms.generalDetails.name." +
+                "validations.duplicate");
+            }
+        };
+
+        /**
+     * Validates the Form.
+     *
+     * @param {FacebookAuthenticationProviderCreateWizardFormValuesInterface} values - Form Values.
+     *
+     * @return {GithubAuthenticationProviderCreateWizardFormErrorValidationsInterface}
+     */
+        const validateForm = (values: OrganizationEnterpriseAuthenticationProviderCreateWizardFormValuesInterface):
+        OrganizationEnterpriseAuthenticationProviderCreateWizardFormErrorValidationsInterface => {
+
+            const errors: OrganizationEnterpriseAuthenticationProviderCreateWizardFormErrorValidationsInterface = {
+                description: undefined,
+                name: undefined
+            };
+
+            if (!values.name) {
+                errors.name = t("console:develop.features.authenticationProvider.forms.common" +
+                ".requiredErrorMessage");
+            }
+            if (!values.description) {
+                errors.description = t("console:develop.features.authenticationProvider.forms.common" +
+                ".requiredErrorMessage");
+            }
+
+            return errors;
+        };
+
+        return (
+            (isIdPListRequestLoading !== undefined && isIdPListRequestLoading === false)
```

Comment:
`isIdPListRequestLoading !== undefined` is this part necessary?

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/3176